api/VsoClient.ts

```diff
@@ -183,41 +183,38 @@ export class VsoClient {
         return url.resolve(this.baseUrl, path.join(this.basePath, relativeUrl));
     }
 
-    private getSerializedObject(queryValue: any, object: any): string {
-        let value:string = "";
-        let first:boolean = true;
-
-        for (let property in object) {
-            if (object.hasOwnProperty(property)) {
-                let prop = object[property];
-                let valueString = this.getValueString(property, prop);
-                if (first && prop !== undefined) {
-                    value += valueString;
-                    first = false;
-                } else if (prop !== undefined) {
-                    value += "&" + valueString;
+    private queryParamsToStringHelper(queryParams: any, prefix: string): string {

```

Comment:
Do you think we need to add more detailed tests for this functionality?

Reply:
Good call, adding another test. This has caused issue so more coverage is definitely better.